web/app/controller/Root.js

```diff
@@ -34,11 +34,17 @@ Ext.define('Traccar.controller.Root', {
     },
 
     onServerReturn: function (options, success, response) {
+        var tokenParameter;
         if (success) {
             Traccar.app.setServer(Ext.decode(response.responseText));
+            tokenParameter = window.location.search.match('token=([a-zA-Z0-9]+)');
             Ext.Ajax.request({
                 scope: this,
                 url: 'api/session',
+                method: 'GET',
+                params: tokenParameter !== null ? {
+                    token: tokenParameter[1]
+                } : {},
```

Comment:
Is formatting OK? Ternary operator is just to not get `api/session?token=` requests